### PR TITLE
Add manual installation instructions

### PR DESCRIPTION
**Pull request by arikmaor**

### CODE REVIEW

File: config/tensorleap.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.cattle.io/v1
```
Comment (roee-shapira-tensorleap) [inline config/tensorleap.yaml:1]: Are you sure that you wanted to add this file to this PR?

**Review by roee-shapira-tensorleap — APPROVED** [review]
LGTM

**Review by yotamazriel — APPROVED** [review]
LGTM

File: config/tensorleap.yaml (hunk 1 above)
Comment (arikmaor) [reply config/tensorleap.yaml:1]: Yes, otherwise I'll have to to inline this in the instructions

**Pull request merged**